src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] array;
+    private int treshold;
+    private int size;
+
+    public MyHashMap() {
+        array = (Node<K,V>[]) new Node[DEFAULT_CAPACITY];
+        treshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + 1 > treshold) {
```

Comment:
```suggestion
        if (size + 1 > treshold) {
```
Вынеси проверку в конец.